src/pokedex.c

```diff
@@ -4169,26 +4182,258 @@ static void PrintMonInfo(u32 num, u32 value, u32 owned, u32 newEntry)
         category = gText_5MarksPokemon;
     }
     PrintInfoScreenText(category, 0x64, 0x29);
-    PrintInfoScreenText(gText_HTHeight, 0x60, 0x39);
-    PrintInfoScreenText(gText_WTWeight, 0x60, 0x49);
+	PrintMonMeasurements(species,owned);
     if (owned)
+        description = GetSpeciesPokedexDescription(species);
+    else
+        description = sExpandedPlaceholder_PokedexDescription;
+    PrintInfoScreenText(description, GetStringCenterAlignXOffset(FONT_NORMAL, description, DISPLAY_WIDTH), 95);
+}
+
+static void PrintMonMeasurements(u16 species, u32 owned)
+{
+    PrintInfoScreenText(gText_HTHeight, 96, 57);
+    PrintInfoScreenText(gText_WTWeight, 96, 73);
+
+    if (owned)
+        PrintOwnedMonMeasurements(species);
+    else
+        PrintUnknownMonMeasurements();
+}
+
+static void PrintUnknownMonMeasurements(void)
+{
+    u8* heightString = GetUnknownMonHeightString();
+    u8* weightString = GetUnknownMonWeightString();
+
+    PrintInfoScreenText(heightString, 129, 57);
+    PrintInfoScreenText(weightString, 129, 73);
+
+	Free(heightString);
+	Free(weightString);
```

Comment:
spaces instead of tabs

Reply:
solved with https://github.com/rh-hideout/pokeemerald-expansion/pull/4183/commits/fd0ab7664345612537c62e1e878263f02568fbfa